### PR TITLE
eth2x-fli-p deploy scripts

### PR DESCRIPTION
**Pull request by ncitron**

### CODE REVIEW

File: polygon/deploy/000_ethfli_system.ts [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+    [ CONTRACT_NAMES.FEE_SPLIT_EXTENSION_NAME ]
+  );
+
+  await protectModule(
+    hre,
+    CONTRACT_NAMES.BASE_MANAGER_NAME,
+    DEBT_ISSUANCE_MODULE_V2,
+    [ CONTRACT_NAMES.FEE_SPLIT_EXTENSION_NAME ]
+  );
+
```
Comment (Alpha-Guy) [inline polygon/deploy/000_ethfli_system.ts:109]: FYI, would need to update SetToken manager to BaseManager contract.

File: polygon/deployments/constants/000_ethfli_system.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { ether } from "@utils/index";
+import { EMPTY_BYTES, ONE_HOUR_IN_SECONDS } from "@utils/constants";
+
+export const CONTRACT_NAMES = {
```
Comment (Alpha-Guy) [inline polygon/deployments/constants/000_ethfli_system.ts:5]: Confirmed parameters with the internal doc. They LGTM.

**Review by Alpha-Guy — CHANGES_REQUESTED** [review]

File: polygon/deploy/000_ethfli_system.ts [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+    hre,
+    CONTRACT_NAMES.FEE_SPLIT_EXTENSION_NAME,
+    CONTRACT_NAMES.BASE_MANAGER_NAME,
+    FEE_SPLIT_EXTENSION.FEE_SPLIT,
+    await findDependency(TREASURY_MULTI_SIG),
+    await findDependency(STREAMING_FEE_MODULE),
+    await findDependency(DEBT_ISSUANCE_MODULE_V2)
+  );
+
+  await addExtension(hre, CONTRACT_NAMES.BASE_MANAGER_NAME, CONTRACT_NAMES.LEVERAGE_EXTENSION_NAME);
```
[the remark this answers is not in the document]
Comment (ncitron) [reply polygon/deploy/000_ethfli_system.ts:94]: `protectModule` automatically adds the extension if it is not already added so no need.

File: polygon/deploy/000_ethfli_system.ts [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+  );
+
+  await protectModule(
+    hre,
+    CONTRACT_NAMES.BASE_MANAGER_NAME,
+    DEBT_ISSUANCE_MODULE_V2,
+    [ CONTRACT_NAMES.FEE_SPLIT_EXTENSION_NAME ]
+  );
+
+
```
[the remark this answers is not in the document]
Comment (ncitron) [reply polygon/deploy/000_ethfli_system.ts:110]: That's the plan

File: polygon/deploy/000_ethfli_system.ts [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+  await polyFillForDevelopment();
+
+  await deploySupplyCapIssuanceHook();
+
+  await deployBaseManager(hre, CONTRACT_NAMES.BASE_MANAGER_NAME, ETHFLI, deployer, dfpMultisigAddress);
+
+  await deployAaveLeverageStrategyExtension();
+
+  await deployFeeExtension(
```
[the remark this answers is not in the document]
Comment (ncitron) [reply polygon/deploy/000_ethfli_system.ts:84]: I am actually planning to update the fee recipient before transferring the set to the manager contract so I can unilaterally change the fee recipient without needing DFP to confirm

**Pull request merged**